src/pages/blogs.js

```diff
@@ -46,6 +46,7 @@ export default () => (
               type="text"
               id="blog-search"
               onChange={e => filterPosts(e)}
+              aria-label="Filter blog posts"
```

Comment:
There's a label in the UI for the input so we don't need this :)

Reply:
Ha sorry 👍  thanks for the reply!